requirements/requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy>=1.18.2
 onnx
-onnxruntime
+onnxruntime_gpu_tensorrt
```

Comment:
Hey @borisfom can you please make sure the dependency on `onnxruntime_gpu_tensorrt` won't break installing/running NeMo applications/tests  on machines WITHOUT GPUs?

Reply:
This is what I have when I pull your PR and try to run ./reinstall.sh on my MacBook:

```
Requirement already satisfied: numpy>=1.18.2 in /Users/tkornuta/opt/anaconda3/envs/nemo-env/lib/python3.7/site-packages (from nemo-toolkit==0.11.0b5) (1.18.4)
Requirement already satisfied: onnx in /Users/tkornuta/opt/anaconda3/envs/nemo-env/lib/python3.7/site-packages/onnx-1.6.0-py3.7-macosx-10.7-x86_64.egg (from nemo-toolkit==0.11.0b5) (1.6.0)
ERROR: Could not find a version that satisfies the requirement onnxruntime_gpu_tensorrt (from nemo-toolkit==0.11.0b5) (from versions: none)
ERROR: No matching distribution found for onnxruntime_gpu_tensorrt (from nemo-toolkit==0.11.0b5)
```

Reply:
Correct - I am working right now to change the version we are building to _gpu (no TRT) and to add onnxruntime build step to x86 build scripts.